projects/dlplibs/Dockerfile

```diff
@@ -32,6 +32,9 @@ ADD https://dev-www.libreoffice.org/corpus/olefuzzer_seed_corpus.zip \
     https://dev-www.libreoffice.org/corpus/pmdfuzzer_seed_corpus.zip \
     https://dev-www.libreoffice.org/corpus/fhfuzzer_seed_corpus.zip \
     $SRC/
+RUN wget \
```

Comment:
@inferno-chromium Because ADD doesn't handle redirections and I don't want to hardcode the use of a particular sf.net mirror.